docs/content/assets/css/customizations.css

```diff
@@ -16,12 +16,16 @@ If you want to append instead, switch `::before` to `::after`.
   src: url('../fonts/external-link.woff') format('woff');
 }
 
-/* Matches the two nav link classes that start with `http` `href` values, regular docs pages use relative URLs instead. */
-.md-tabs__link[href^="http"]::before, .md-nav__link[href^="http"]::before {
+/* 
+   Since mkdocs-material 9.5.5 broke support in our docs from DMS v13.3.1, we now use our own class name,
+   which has been included for the two external nav links in mkdocs.yml via workaround (insert HTML).
+*/
```

Comment:
Praise be! 😨 🙏 

```suggestion
/*
  Since mkdocs-material 9.5.5 broke support in our docs from DMS v13.3.1, we now use our own class name,
  which has been included for the two external nav links in mkdocs.yml via workaround (insert HTML).
*/
```